python/gui/src/hetsgui/windows/ConservativityCheckWindow.py

```diff
@@ -0,0 +1,129 @@
+import logging
+import threading
+import traceback
+import typing
+
+from gi.repository import Gtk, GLib
+
+from ..GtkSmartTemplate import GtkSmartTemplate
+from hets import DevGraphEdge, ConsistencyChecker, ConservativityChecker, ConsistencyKind, TheoremDevGraphEdge
+from ..widgets import GridWithToolComorphismSelector
+
+
+@GtkSmartTemplate
+class ConservativityCheckWindow(Gtk.Window):
+    __gtype_name__ = 'ConservativityCheckWindow'
+    _logger = logging.getLogger(__name__)
+
+    _btn_check: Gtk.Button = Gtk.Template.Child()
+
+    _checker_model: Gtk.ListStore = Gtk.Template.Child()
+    _combo_checker: Gtk.ComboBox = Gtk.Template.Child()
+    _lbl_status: Gtk.Label = Gtk.Template.Child()
+    _lbl_output: Gtk.Label = Gtk.Template.Child()
+
+    @property
+    def selected_conservativity_checker(self) -> typing.Optional[ConservativityChecker]:
+        active_index = self._combo_checker.get_active()
+        cc_name = self._checker_model[active_index][0] if active_index >= 0 else None
+        cc = self._edge.get_conservativity_checker_by_name(cc_name) if cc_name else None
+        return cc
+
+    def __init__(self, edge: DevGraphEdge, **kwargs):
+        super().__init__(title=f"Check conservativity of {edge.title()}", **kwargs)
+
+        self.checking_thread = None
+        self._edge = edge
+
+        self._update_status(edge.conservativity())
+
+        ccs = self._edge.get_usable_conservativity_checkers()
+        for cc in ccs:
+            self._checker_model.append([cc.name(), cc.name(), 0])
+
+        if len(self._checker_model) > 0:
+            self._combo_checker.set_active(0)
+
+    @Gtk.Template.Callback()
+    def on_check_clicked(self, *args):
+        self.checking_thread = threading.Thread(target=self._check_consistency)
+        # self.proving_thread.daemon = True
+        self.checking_thread.start()
+
+    def _update_status(self, status: typing.Union[ConsistencyKind, str]):
+        style_context = self._lbl_status.get_style_context()
+        style_context.remove_class("consistency-kind-inconsistent")
```

Comment:
where is "consistency-kind-inconsistent" added? I can only see it being removed, and then css declarations.